MvvmCross/Core/Core/ViewModels/MvxViewModelExtensions.cs

```diff
@@ -32,6 +32,10 @@ public static void CallBundleMethods(this IMvxViewModel viewModel, string method
         public static void CallBundleMethod(this IMvxViewModel viewModel, MethodInfo methodInfo, IMvxBundle bundle)
         {
             var parameters = methodInfo.GetParameters().ToArray();
+
+            if (bundle == null && parameters.Count() > 0)
```

Comment:
I think a comment is needed right above the `if` switch... something like "Make sure we have a bundle that matches function parameters" or whatever.  It just seems to me like we're getting in this function when we shouldn't be.